samples/system-test/spanner.test.js

```diff
@@ -66,6 +66,17 @@ const GAX_OPTIONS = {
   },
 };
 
+const delay = async test => {
+  const retries = test.currentRetry();
+  if (retries === 0) return; // no retry on the first failure.
+  // see: https://cloud.google.com/storage/docs/exponential-backoff:
+  const ms = Math.pow(2, retries) * 250 + Math.random() * 1000;
```

Comment:
Out of curiosity, why is it not?
```
const ms = (Math.pow(2, retries) + Math.random()) * 1000;
```

I guess I'm wondering about the 250msec increments of time.

Reply:
Done